lib/calculator_page.dart

```diff
@@ -10,147 +11,94 @@ class CalculatorPage extends StatefulWidget {
 }
 
 class _CalculatorPageState extends State<CalculatorPage> {
-  var userQuestion = '';
-  var userAnswer = '';
-
-  final List<String> buttons = [
-    'C',
-    'DEL',
-    '%',
-    '/',
-    '9',
-    '8',
-    '7',
-    'x',
-    '6',
-    '5',
-    '4',
-    '-',
-    '3',
-    '2',
-    '1',
-    '+',
-    '0',
-    '.',
-    'ANS',
-    '=',
-  ];
+  
+ 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.deepPurple[100],
-      body: Column(
-        children: [
-          Expanded(
-            child: Container(
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                children: [
-                  const SizedBox(
-                    height: 50,
-                  ),
-                  Container(
-                    padding: const EdgeInsets.all(20),
-                    alignment: Alignment.centerLeft,
-                    child: Text(
-                      userQuestion,
-                      style: const TextStyle(fontSize: 20),
+     CalculatorProvider calculate =
+      Provider.of<CalculatorProvider>(context);
+    return Consumer<CalculatorProvider>(builder: (context, value, child) {
```

Comment:
you can use value vairable instead of calculate variable on line 18

Reply:
done